gui/default/assets/lang/lang-en.json

```diff
@@ -257,6 +261,7 @@
    "This is a major version upgrade.": "This is a major version upgrade.",
    "This setting controls the free space required on the home (i.e., index database) disk.": "This setting controls the free space required on the home (i.e., index database) disk.",
    "Time": "Time",
+   "Time when this item has been last modified": "Time when this item has been last modified",
```

Comment:
This string, and the one about "device which has done the latest modification" both sound like broken or at least non-idiomatic English. Though I can't off hand pinpoint issue - spider sense tingling. Anyone native speaking here? (sorry if that's you @imsodin and I'm barking up the wrong tree)

Reply:
I am Swiss - no trace of native English here :)

Reply:
I tried to come up with alternatives, but they are all worse than this - so unless suddenly someone with an English language intuition turns up, we may have to go with this?

Reply:
Doesn't trigger my spider senses.

Reply:
"Last modification time", "Time the item was last modified" sound better to me.

(We can bike shed the wording a bit because it sort of gets set in stone due to translations)

Reply:
The first one sounds wrongish to me, but the second is nicer (shorter) than mine and google agrees with your tense ("was last modified" is much more common than "has been last modified" - very scientific decision method). So that's settled then :)